pkg/plugin/aggregation/aggregator.go

```diff
@@ -244,6 +282,30 @@ func (a *Aggregator) HandleHTTPResult(result *plugin.Result, w http.ResponseWrit
 	}
 }
 
+// HandleHTTPStatus wraps the aggregators processStatus method in such a way as to respond
+// with appropriate logging and HTTP codes.
+func (a *Aggregator) HandleHTTPStatus(status plugin.StatusUpdate, w http.ResponseWriter) {
```

Comment:
Will add a processstatus test already; may want to add a test here to ensure we properly continue to report the errors/codes as intended.